server/views/components/tags/index.njk

```diff
@@ -0,0 +1,5 @@
+<ul class="tags">
+  {% for tag in tags %}
+    <li class="tags__item"><a class="tags__link" href="{{ tag.url }}">{{ tag.text }}</a>{{ ',' if not loop.last }}</li>
```

Comment:
Probably, yep – I'm getting away without the space here because of the line-break with inline-block. Will fix.